### PR TITLE
Datastore read event

### PR DESCRIPTION
**Pull request by ChengyuanZhao**
related #1113 
datastore read and query events

### CODE REVIEW

File: docs/src/main/asciidoc/datastore.adoc [hunk 1]
```diff
@@ -1191,6 +1191,8 @@ Spring Data Cloud Datastore publishes events extending the Spring Framework's `A
 [options="header"]
 |===
 | Type | Description | Contents
+| `AfterKeyReadEvent`|Published immediately after read by-key operations are executed by `DatastoreTemplate`| The entities read from Cloud Datastore and the original keys in the request.
```
Comment (meltsufin) [inline docs/src/main/asciidoc/datastore.adoc]: The name of the event sounds like you're reading a key.
Maybe `AfterReadByKeyEvent` or `AfterFindByKeyEvent`?

File: docs/src/main/asciidoc/datastore.adoc [hunk 2]
```diff
@@ -1191,6 +1191,8 @@ Spring Data Cloud Datastore publishes events extending the Spring Framework's `A
...
+| `AfterQueryReadEvent`|Published immediately after read byquery operations are executed by `DatastoreTemplate`| The entities read from Cloud Datastore and the original query in the request.
```
Comment (meltsufin) [inline docs/src/main/asciidoc/datastore.adoc]: Here also, I would consider calling it `AfterReadByQueryEvent` or just simply `AfterQueryEvent`.

Comment (ChengyuanZhao) [reply docs/src/main/asciidoc/datastore.adoc]: Yea can do

File: docs/src/main/asciidoc/datastore.adoc (hunk 1 above)
Comment (ChengyuanZhao) [reply docs/src/main/asciidoc/datastore.adoc]: Can do

**Pull request merged**